components/FilterForm/FilteredCarsList.js

```diff
@@ -20,6 +28,23 @@ function FilteredCarsList({ list }) {
         <StyledList>
           {list.map((car) => (
             <li key={car.id}>
+              <StyledLikeButton onClick={() => handleToggleFavorite(car.id)}>
+                {car.isFavorite ? (
+                  <Image
+                    src={"/heart.png"}
+                    alt="liked"
+                    width={40}
+                    height={40}
+                  ></Image>
+                ) : (
+                  <Image
+                    src={"/notLiked.png"}
+                    alt="notLiked"
+                    width={40}
+                    height={40}
+                  ></Image>
+                )}
+              </StyledLikeButton>
```

Comment:
You could think about creating a new component here as you use this part for your Favorites and your FilteredCarsList. I would highly recommend to do this. ;)